### PR TITLE
gr-digital: Un-break Constellation Object and Constellation Rect Object

### PR DESCRIPTION
**Pull request by haakov**
Fixes https://github.com/gnuradio/gnuradio/issues/2050

### CODE REVIEW

File: gr-digital/grc/digital_constellation.block.yml [hunk 1]
```diff
@@ -49,9 +49,9 @@ templates:
         % else:
         self.${id} = ${id} = digital.constellation_${type}().base()
         % endif
-        % if str(soft_dec_lut).lower() == ''"auto"'' or str(soft_dec_lut).lower() == "''auto''":
+        % if str(soft_dec_lut).lower() == "auto" or str(soft_dec_lut).lower() == "auto":
```
Comment (argilo) [inline gr-digital/grc/digital_constellation.block.yml]: This doesn't look right. It's the same comparison twice in a row.

File: gr-digital/grc/digital_constellation_rect.block.yml [hunk 2]
```diff
@@ -49,9 +49,9 @@ templates:
     var_make: |-
         self.${id} = ${id} = digital.constellation_rect(${const_points}, ${sym_map},
         ${rot_sym}, ${real_sect}, ${imag_sect}, ${w_real_sect}, ${w_imag_sect}).base()
-        % if str(soft_dec_lut).lower() == ''"auto"'' or str(soft_dec_lut).lower() == "''auto''":
+        % if str(soft_dec_lut).lower() == "auto" or str(soft_dec_lut).lower() == "auto":
```
Comment (argilo) [inline gr-digital/grc/digital_constellation_rect.block.yml]: Same here.

File: gr-digital/grc/digital_constellation.block.yml (hunk 1 above)
Comment (haakov) [reply gr-digital/grc/digital_constellation.block.yml]: Haha wow you're right

**Comment by haakov** [comment]
Okay, this makes more sense :)

**Comment by argilo** [comment]
The code looks like it probably reproduces the behaviour of the old template now (although I haven't tested whether it generates the right Python code).

It seems like it would be better to just use `soft_dec_lut()` though to access the underlying Python object (once that's working properly), instead of working with the string version produced by `repr()`.

I'm okay leaving the code as it is though, assuming it works properly.

**Pull request merged**